dashboard/app/views/devise/registrations/new.html.haml

```diff
@@ -54,7 +54,8 @@
             .error_in_field
 
         #name-block.itemblock
-          .labelblock= t('activerecord.attributes.user.name')
+          #name-student.labelblock= t('activerecord.attributes.user.name_example')
```

Comment:
After adding the break tag you may need to do `labelblock!= t(` in order to allow the HTML.